.github/workflows/readme.yml

```diff
@@ -0,0 +1,40 @@
+name: Build README
+
+on:
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: '36 8 * * *'
+    - cron: '36 14 * * *'
+    - cron: '36 21 * * *'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+    - uses: actions/cache@v3
+      name: Configure pip caching
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Install Python dependencies
+      run: |
+        python -m pip install -r requirements.txt
+    - name: Update README
+      run: |-
+        python dynamic_readme.py
+        cat README.md
+    - name: Commit and push if README changed
+      run: |-
+        git diff
+        git config --global user.name "${{ env.CI_COMMIT_AUTHOR }}"
+        git config --global user.email "username@users.noreply.github.com"
```

Comment:
@addyess ah...this wants an actual username. D'oh! - that will teach me for just accepting your changes

Reply:
yeah i'm so sorry... i just gave you a really crappy suggestion because i'm incapable of reading slowly